scripts/standard/run-tests.sh

```diff
@@ -1,9 +1,11 @@
 #!/bin/bash
 
+TIMESTAMP=`date '+%Y-%m-%d_%H:%M:%S'`
+
 EXEDIR=${1:-"../../build/bin"}
 DATADIR=${2:-"/data/gunrock_dataset/large"}
 DEVICE=${3:-"0"}
-TAG=${4:-"V100"}
+TAG=${4:-"$TIMESTAMP"}
```

Comment:
This is a good idea. I presume the same tag goes to every experiment in the same run. I like it.